packages/client/utils/jpgWithoutEXIF.ts

```diff
@@ -7,9 +7,13 @@ const jpgWithoutEXIF = (file: File) => {
       canvas.width = img.naturalWidth
       canvas.height = img.naturalHeight
       context.drawImage(img, 0, 0, canvas.width, canvas.height)
-      canvas.toBlob((blob) => {
-        resolve(blob!)
-      })
+      canvas.toBlob(
+        (blob) => {
+          resolve(blob!)
+        },
+        'image/jpeg',
+        0.75
```

Comment:
ahhh that's the ticket!
curious, did you compare the input img size vs. output image size after this change?

Reply:
Yep! Input vs output size of files around the 1mb limit would be very similar. Some of the smaller files would vary a bit more but still be pretty similar